.travis.yml

```diff
@@ -27,11 +19,33 @@ branches:
 matrix:
     fast_finish: true
     include:
+          # Minimum supported PHP and Symfony version
         - php: 5.4
-          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" COVERAGE=true TEST_COMMAND="composer test-ci"
-
+          env: DEPENDENCIES="minimum" COVERAGE=true TEST_COMMAND="composer test-ci"
+
+          # Test the latest stable release
+        - php: 7.1
+
+          # Test LTS versions
+        - php: 7.1
+          env: DEPENDENCIES="symfony/lts:v2"
+        - php: 7.1
+          env: DEPENDENCIES="symfony/lts:v3"
+
+          # Latest beta release
```

Comment:
I suggest testing against dev rather than beta (especially if you allow them to fail):

- if something breaks in an early Symfony 4.1 version, you may have 5 months to report it or adapt, instead of a few weeks/days
- most of your other deps don't release beta at all, so you will not catch anything until it reaches other jobs too